packages/application-shell/src/components/navbar/navbar.spec.js

```diff
@@ -1,1293 +1,4 @@
-            });
-            it('should set isExpanderVisible to true', () => {
-              expect(wrapper).toHaveState('isExpanderVisible', true);
-            });
-          });
-        });
-      });
-    });
-    describe('handleToggleItem', () => {
-      describe('if activeItemIndex is not the same as the given index', () => {
-        beforeEach(() => {
-          wrapper.setState({ activeItemIndex: 'fixed-1' });
-          wrapper.instance().handleToggleItem('fixed', 0);
-        });
-        it('should update activeItemIndex with the new index', () => {
-          expect(wrapper).toHaveState('activeItemIndex', 'fixed-0');
-        });
-      });
-    });
-    describe('shouldCloseMenuFly', () => {
-      describe('if node does not contain event target and menu is not open', () => {
-        beforeEach(() => {
-          wrapper.setProps({ rootNode: { contains: () => false } });
-          wrapper.setState({ isMenuOpen: false });
-          wrapper
-            .instance()
-            .shouldCloseMenuFly({ target: 'foo', type: 'mouseleave' });
-        });
-        it('should unset activeItemIndex', () => {
-          expect(wrapper).toHaveState('activeItemIndex', null);
-        });
-      });
-      describe('if menu is open and event is a mouseleave', () => {
-        beforeEach(() => {
-          wrapper.setProps({ rootNode: { contains: () => true } });
-          wrapper
-            .instance()
-            .shouldCloseMenuFly({ target: 'foo', type: 'mouseleave' });
-        });
-        it('should unset activeItemIndex', () => {
-          expect(wrapper).toHaveState('activeItemIndex', null);
-        });
-      });
-    });
-    describe('handleToggleMenu', () => {
-      describe('if menu is open and activeItemIndex is not null', () => {
-        beforeEach(() => {
-          wrapper.setState({ isMenuOpen: true, activeItemIndex: 'fixed-0' });
-          wrapper.instance().handleToggleMenu();
-        });
-        it('should unset activeItemIndex', () => {
-          expect(wrapper).toHaveState('activeItemIndex', null);
-        });
-      });
-      describe('if menu is open', () => {
-        beforeEach(() => {
-          wrapper.setState({ isMenuOpen: true });
-          wrapper.instance().handleToggleMenu();
-        });
-        it('should update isForcedMenuOpen to false', () => {
-          expect(window.localStorage.setItem).toHaveBeenCalledWith(
-            STORAGE_KEYS.IS_FORCED_MENU_OPEN,
-            false
-          );
-        });
-      });
-      describe('if menu is not open', () => {
-        beforeEach(() => {
-          wrapper.setState({ isMenuOpen: false });
-          wrapper.instance().handleToggleMenu();
-        });
-        it('should update isForcedMenuOpen to true', () => {
-          expect(window.localStorage.setItem).toHaveBeenCalledWith(
-            STORAGE_KEYS.IS_FORCED_MENU_OPEN,
-            true
-          );
-        });
-      });
-    });
-  });
-});
```

Comment:
Nostalgia: I spent so much time bolting e.g. permissions and menu visibilities in here 🐼